### PR TITLE
improve accessibility and add labels

### PR DESCRIPTION
**Pull request by tcase360**

### CODE REVIEW

File: src/views/index.html [hunk 1]
```diff
@@ -29,37 +29,60 @@
...
+
+            {% for entity in entities %}
+            <label for="{{ entity.query_param }}">
+                {{ entity.entity_name }}
+                <input
+                    id="{{ entity.query_param }}"
+                    class="entity-input"
+                    type="text"
+                    name="{{ entity.query_param }}"
+                    aria-label="{{ entity.entity_name }}"
```
Comment (sarayourfriend) [inline src/views/index.html]: FWIW aria-label is redundant here given the element is wrapped in a label. I think even the `for` attribute on the label is redundant.

**Review by sarayourfriend — COMMENTED** [review]
LGTM so far! I love the fieldset. Only problem I see is that the contrast between the fieldset outline and the background is pretty narrow, it even hurts my poor eyes 😭 Could we update it so the fieldset outline is just a regular black line?

**Review by mpuckett159 — APPROVED** [review]
Looks good, makes the form look much better.

**Review by AetherUnbound — APPROVED** [review]
TIL about `fieldset`! How cool!

File: src/views/index.html (hunk 1 above)
Comment (tcase360) [reply src/views/index.html]: Ok, you're definitely right for the `aria-label` - I did read somewhere though that the `for` should be present regardless. I'll take the `aria-label` out.

As for the `fieldset`, is the suggestion to make a bigger border with a darker color?

Comment (sarayourfriend) [reply src/views/index.html]: Yeah just something with clearer contrast, right now the default user agent styles use some kind of gray bevel that clashes with the blue in an unpleasant way.

**Review by sarayourfriend — APPROVED** [review]
Awesome! 🚀

**Pull request merged**